app/src/preHome.js

```diff
@@ -0,0 +1,163 @@
+import React, { useState, useEffect, useCallback } from 'react';
+import Button from './components/Button/Button';
+import FormInput from "./components/FormInput/FormInput";
+import { BrowserRouter as Router, useHistory } from "react-router-dom";
+import { API, Auth } from "aws-amplify";
+import s from "./preHome.module.css";
+
+
+export default function PreHome() {
+    const [isLoading, setIsLoading] = useState(false);
+    const [searchVal, setSearchVal] = useState("");
+    const history = useHistory();
+
+    const toRecordingPage = () => history.push("/recording");
+
+    const handleSubmit = event => {
+        event.preventDefault();
+        alert(`Submitting Name ${searchVal}`)
```

Comment:
Do we need this alert?

Reply:
Removed. https://github.com/aws-samples/medical-transcription-analysis/blob/8b46704678a936967c21d5a45384e3f84615c38f/app/src/preHome.js#L24-L26